python/ray/projects/scripts.py

```diff
@@ -303,23 +336,25 @@ def stop(name):
 def session_start(command, args, shell, name):
     runner = SessionRunner(session_name=name)
```

Comment:
Do all the sessions get the same name?

Reply:
I'm now creating a unique name for them :)